server/src/main/java/org/apache/kafka/server/config/KRaftConfigs.java

```diff
@@ -114,6 +114,10 @@ public class KRaftConfigs {
     public static final String SERVER_MAX_STARTUP_TIME_MS_DOC = "The maximum number of milliseconds we will wait for the server to come up. " +
             "By default there is no limit. This should be used for testing only.";
 
+    public static final String MIN_SLOW_EVENT_TIME_MS_CONFIG = "controller.slow.event.min.ms";
+    public static final int MIN_SLOW_EVENT_TIME_MS_DEFAULT = 200;
+    public static final String MIN_SLOW_EVENT_TIME_MS_DOC = "Do not log slow controller events which are slowed than this duration.";
```

Comment:
```suggestion
    public static final String MIN_SLOW_EVENT_TIME_MS_DOC = "Do not log slow controller events which are slower than this duration.";
```

Something like this could be a bit easier to understand:

> Log controller events take longer than this duration.

I'd also add reference the P99 threshold, e.g.

> Log controller events with a 30 second P99 duration that is longer than this minimum.